x-pack/plugins/dataset_quality/public/components/dataset_quality/summary_panel/datasets_quality_indicators.tsx

```diff
@@ -0,0 +1,109 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+import {
+  EuiFlexGroup,
+  EuiPanel,
+  EuiFlexItem,
+  EuiTitle,
+  EuiText,
+  EuiHealth,
+  EuiIconTip,
+  EuiSkeletonTitle,
+} from '@elastic/eui';
+import { useSummaryPanelContext } from '../../../hooks';
+import {
+  summaryPanelQualityDegradedText,
+  summaryPanelQualityGoodText,
+  summaryPanelQualityPoorText,
+  summaryPanelQualityText,
+  summaryPanelQualityTooltipText,
+} from '../../../../common/translations';
+import { mapPercentagesToQualityCounts } from '../../quality_indicator';
+
+export function DatasetsQualityIndicators() {
+  const { datasetsQuality, isDatasetsQualityLoading } = useSummaryPanelContext();
+  const qualityCounts = mapPercentagesToQualityCounts(datasetsQuality.percentages);
+
+  return (
+    <EuiPanel hasBorder>
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexGroup alignItems="center" gutterSize="xs" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <EuiText size="s">{summaryPanelQualityText}</EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiIconTip content={summaryPanelQualityTooltipText} />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiFlexGroup gutterSize="m" alignItems="flexEnd">
+          <QualityIndicator
+            value={qualityCounts.poor}
+            quality="danger"
+            description={summaryPanelQualityPoorText}
+            isLoading={isDatasetsQualityLoading}
+          />
+          <VerticalRule />
+          <QualityIndicator
+            value={qualityCounts.degraded}
+            quality="warning"
+            description={summaryPanelQualityDegradedText}
+            isLoading={isDatasetsQualityLoading}
+          />
+          <VerticalRule />
+          <QualityIndicator
+            value={qualityCounts.good}
+            quality="success"
+            description={summaryPanelQualityGoodText}
+            isLoading={isDatasetsQualityLoading}
+          />
+        </EuiFlexGroup>
+      </EuiFlexGroup>
+    </EuiPanel>
+  );
+}
+
+const QualityIndicator = ({
+  value,
+  quality,
+  description,
+  isLoading,
+}: {
+  value: number;
+  quality: 'success' | 'danger' | 'warning';
+  description: string;
+  isLoading: boolean;
+}) => {
+  return (
+    <EuiFlexGroup direction="column" gutterSize="xs">
+      {isLoading ? (
+        <EuiSkeletonTitle size="m" />
+      ) : (
+        <EuiTitle size="m">
+          <h3>
+            <EuiHealth textSize="inherit" color={quality}>
+              {value || 0}
+            </EuiHealth>
+          </h3>
+        </EuiTitle>
+      )}
+      <EuiText color={quality}>
+        <h5>{description}</h5>
+      </EuiText>
+    </EuiFlexGroup>
+  );
+};
+
+const VerticalRule = styled.span`
```

Comment:
What is the preferred way of styling? Is it styled components or through the css prop from `@emotion/react`? (https://eui.elastic.co/#/guidelines/getting-started#styling-your-application)